classes/cave.js

```diff
@@ -542,10 +443,98 @@ class Cave {
 
             if (reaction.emoji.name === this.adminEmojis.first().name) {
                 await this.newRole(adminConsole, promptUserId);
+
+                // let admin know the action was succesfull
+                adminConsole.send('<@' + promptUserId + '> The role has been added!').then(msg => msg.delete({ timeout: 8000 }));
+            } else if (reaction.emoji.name === Array.from(this.adminEmojis.keys())[1]) {
+                let all = await Prompt.yesNoPrompt('Type "yes" if you would like to delete all mentor channels, "no" if you would like to only delete some.', adminConsole, promptUserId);
+                if (all) {
+                    let deleteNow = await Prompt.yesNoPrompt('All ticket categories older than a minute will be deleted. Type "yes" to confirm or "no" to set a different timeframe. Careful - this cannot be undone!', adminConsole, promptUserId);
+                    var age;
+                    adminConsole.guild.channels.cache.forEach(async channel => {
+                        (deleteNow) ? age = 1 : age = await Prompt.numberPrompt('Enter the number of minutes. All ticket channels older than this time will be deleted. Careful - this cannot be undone!', adminConsole, promptUserId);
```

Comment:
Is it intentional to ask the age inside the forEach loop? I think this should be outside?

Reply:
yes i'm dumb dumb